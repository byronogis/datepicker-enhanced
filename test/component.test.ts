import { describe, expect, it } from 'vitest'
import { mount } from '@vue/test-utils'
import MyComponent from '../src/MyComponent.vue'

describe('my vue component', () => {
  it('should render', () => {
    const wrapper = mount(MyComponent, {
      props: {
        name: 'byronogis',
      },
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
