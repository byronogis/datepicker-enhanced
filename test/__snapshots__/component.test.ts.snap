// Vitest Snapshot v1

exports[`my vue component > should render 1`] = `"<div> Hello, this is byronogis</div>"`;
